packages/rancher-windows-gmsa/charts/templates/clusterrolebinding.yaml

```diff
@@ -0,0 +1,16 @@
+# bind that role to the webhook's service account
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: {{ .Release.Name }}
+  labels:
+    chart: {{ template "gmsa.chartref" . }}

```

Comment:
Why not just include the key `chart` in the template as well? So you can just call this as:

```yaml
  labels: {{ template "gmsa.chartref" . | nindent 4 }}
```